shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
I'm talking only about the cartesian_config variants, not about the `machine_type` value. So yes, `machine_type` has to be `s390-ccw-virtio`, but the variant can be called `s390-virtio` and the arch could be named `s390x`.